jetty-core/jetty-util/src/test/java/org/eclipse/jetty/util/BlockingTest.java

```diff
@@ -36,108 +34,84 @@ public class BlockingTest
     @Test
     public void testRunBlock() throws Exception
     {
-        long start;
         try (Blocker.Runnable runnable = Blocker.runnable())
         {
             runnable.run();
-            start = NanoTime.now();
             runnable.block();
```

Comment:
Is there some awaitility for "run this and wait at most X to complete"?

Reply:
Not in awaitility, but a test can be annotated with `@Timeout`. Let me add it.